README.rst

```diff
@@ -1,205 +1,186 @@
 Papis
 =====
 
-|Build_Status| |ghbadge| |Coveralls| |RTD|
-|Code_Quality|
-|PyPI-Versions| |MYPY-CHECKED| |PyPI-Downloads|
-|Contributors| |zenodo_badge| |OpenHub|
+|Build_Status| |ghbadge| |Coveralls| |RTD| |Code_Quality| |zenodo_badge|
 
+Papis is a powerful and highly extensible CLI document and bibliography manager.
 
-|Pypi|
-|Packaging_status|
+|first_glance|
 
-|zulip|
-|libera|
+With Papis, you can search your library for books and papers, add documents and notes, import and export to and from other formats, and much much more. Papis uses a human-readable and easily hackable `.yaml` file to store each entry's bibliographical data. It strives to be easy to use while providing a wide range of features. And for those who still want more, Papis makes it easy to write scripts that extend its features even further.
 
-Description
------------
+Features
+--------
 
-Papis is a powerful and highly extensible command-line based document
-and bibliography manager.
+- **Add documents** and automatically fetch metadata with *crossref*.
```

Comment:
```suggestion
- **Add documents** and automatically fetch metadata, e.g. with *crossref*.
```
? Since it also supports fetching metadata with `isbnlib`, from pubmed and just scrapping web pages.

Reply:
Oh that is true! I just copied it from the old readme. I think we can simply delete the mention of crossref. This level of detail isn't needed for this high-level overview of functionality.

Reply:
Yep, sounds good!